pages/story/story_data.js

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
should we rename this file to `data` as the folder name is already `story`?

Reply:
can we put the contents from stories_data file in a json file for now? and maybe add this data file under pages/api with a simple function to parse the json and give out an stories_array. How are we planning to fetch this data in general?

Reply:
or maybe another folder data, in the bphc-ulti-web folder

Reply:
yep data folder works.